charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -14,7 +14,26 @@ spec:
   - Ingress
   ingress:
   - from:
+      {{- with .Values.networkPolicy }}
+      namespaceSelector:
+        matchExpressions:
+          {{- toYaml .namespaceExpressions | nindent 8 }}
+        matchLabels:
+          {{- range $key, $value := .namespaceLabels }}
+          {{ $key | quote }}: {{ $value | quote }}
+          {{- end }}
+      podSelector:
+        matchExpressions:
+          {{- toYaml .podExpressions | nindent 8 }}
+        matchLabels:
+          {{- range $key, $value := .podLabels }}
+          {{ $key | quote }}: {{ $value | quote }}
+          {{- end }}
+      {{- end }}
     ports:
     - protocol: TCP
       port: 9443 # webhook access
+  # Allow prometheus scrapes for metrics
+  - ports:
+      - port: 8000
```

Comment:
We should test `metricsService.create` to know if we have to allow this port. Something like:

```yaml
{{- if .Values.metricsService.create }}
- ports:
    - port: 8000
{{- end }}
```

Reply:
Instead hardcoding the port should we use `metricsService.port`?
```
{{- if .Values.metricsService.create }}
- ports:
    - port: {{ .Values.metricsService.port }}
{{- end }}
```

Reply:
Yes that would be nice to have.